sky/tools/create_xcframework.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
```

Comment:
nit: google python style guide requires some future imports:
https://google.github.io/styleguide/pyguide.html#from-__future__-imports

Reply:
Not sure what the minimum python version we support (I have 2.7.16 on my machine), I ran it through `futurize`.
```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

import argparse
import errno
import os
import shutil
import subprocess
import sys
```
This matches the other places we use `from __future__ import`
https://github.com/flutter/engine/blob/8f060b99547944606c9f5a7b1f0daea13b94f86e/build/copy_info_plist.py#L16-L23
https://github.com/flutter/engine/blob/b7e5940e3b1859a3dcd9aac86d3ac11770de8aac/build/git_revision.py#L7-L15